src/runtime_src/core/include/xrt_error_code.h

```diff
@@ -64,7 +64,25 @@
  *
  */
 typedef uint64_t xrtErrorCode;
- 
+
+#define XRT_ERROR_NUM_MASK		0xFFFFUL
+#define XRT_ERROR_NUM_SHIFT		0
+#define XRT_ERROR_DRIVER_MASK		0xFUL
+#define XRT_ERROR_DRIVER_SHIFT		16
+#define XRT_ERROR_SEVERITY_MASK		0xFUL
+#define XRT_ERROR_SEVERITY_SHIFT	24
+#define XRT_ERROR_MODULE_MASK		0xFUL
+#define XRT_ERROR_MODULE_SHIFT		32
+#define XRT_ERROR_CLASS_MASK		0xFUL
+#define XRT_ERROR_CLASS_SHIFT		40
+
+#define	XRT_BUILD_ERROR_CODE(num, driver, severity, module, eclass) \
```

Comment:
While yours read better, I think we need the verb at the end to be consistent.  So let's call it XRT_ERROR_CODE_BUILD.  Part of the reason for this naming convention is that it makes it fairly clear which header declares the API.

Reply:
Done.